packages/core/src/index.ts

```diff
@@ -858,6 +858,10 @@ class Hopr extends EventEmitter {
     return await ethereum.getPublicKeyOf(addr)
   }
 
+  public async signArbitraryMessage(message: Uint8Array) {
```

Comment:
Sign *arbitrary* is highly misleading. Furthermore, a signing method that just signs everything without demanding any structure is equivalent to exposing the private key.

Pleasw change the method name and add a prefix, e.g. `HOPR signed message`